### DISCUSSION TITLE
Why does UVR need a good GPU?

### DISCUSSION THREAD

**Discussion by OnivalSouza** [Anjok07/ultimatevocalremovergui#1634] [Q&A] [answered]
Why does UVR need a good GPU? Doesn't it use the CPU? My GPU is integrated, which is very weak.
I heard that a graphics card is only for watching videos. Why does UVR need it?
I'm inexperienced in these matters!
Thanks!

**Comment by zdd123456M** [comment]
When will the version be updated to 5.7

**Reply by OnivalSouza** [reply]
> Quando a versão será atualizada para 5.7

Please, I asked a question and you answer with another question?
Please create a new discussion and ask your question!
Thank you!
*
Por favor, eu fiz uma pergunta e você responde com outra pergunta?
Crie uma nova discussão e coloque sua dúvida, por favor!
Obrigado! (pt-br)

**Comment by zdd123456M** [comment]
Yes, when will it be updated to version 5.7



---Original---
From: ***@***.***&gt;
Date: Mon, Nov 25, 2024 11:43 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [Anjok07/ultimatevocalremovergui] Why does UVR need a good GPU?(Discussion #1634)




  
Quando a versão será atualizada para 5.7
  
Please, I asked a question and you answer with another question?
 Please create a new discussion and ask your question!
 Thank you!
 *
 Por favor, eu fiz uma pergunta e você responde com outra pergunta?
 Crie uma nova discussão e coloque sua dúvida, por favor!
 Obrigado! (pt-br)

**Comment by Lolen10** [comment, marked as answer]
Ultimate Vocal Remover uses a special kind of technology called machine learning to separate vocals from instruments in songs. This is a very complex task because it involves analyzing the audio file in great detail and performing millions of calculations to figure out what’s a voice and what’s an instrument.

Your computer has two main parts that can handle these kinds of tasks: the CPU and the GPU. The CPU (Central Processing Unit) is like the brain of your computer, and it’s designed to handle a wide variety of tasks. However, the GPU (Graphics Processing Unit) is built specifically to handle large numbers of similar calculations **all at once**, which makes it perfect for tasks like video processing, gaming, and, in this case, separating audio.

Even though GPUs are often thought of as being for watching videos or playing games, they’re also very good for machine learning tasks like the ones UVR uses. If your GPU is integrated (meaning it’s built into your CPU rather than being a separate piece of hardware), it’s usually much weaker than a dedicated GPU (a separate graphics card). Integrated GPUs share resources with your CPU, which means they’re not as powerful, and UVR may take much longer to process audio or might not work as well.

While UVR can still work with just the CPU, it will run significantly slower because the CPU isn’t optimized for this kind of heavy calculation. A dedicated GPU can speed things up a lot, making UVR process your files faster and more efficiently.

BTW: You can just use the CPU by just unchecking this box: 
![grafik](https://github.com/user-attachments/assets/92351da0-61ad-4547-a4e4-622b543513bc)

but I doubt that this will be any faster.

**Reply by OnivalSouza** [reply]
Perfect answer!
Exactly what I wanted to know!
Thank you very much!
Note: - The "GPU" option is always disabled here on my PC.
I'm using MVSEP online and it's perfect!
